src/plugins/intel_cpu/src/nodes/kernels/stack_allocator.hpp

```diff
@@ -0,0 +1,474 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <string>
+#include <vector>
+#include <ie/ie_common.h>
+#include <cpu/x64/jit_generator.hpp>
+
+namespace ov {
+namespace intel_cpu {
+
+namespace x64 = dnnl::impl::cpu::x64;
```

Comment:
As it was discussed before, generally this no a very good practice to declare nay namespace usage in header file. In this particular situation, we can easily resolve the issue first just add type alias to `StackAllocator` class:
```cpp
class StackAllocator final {
public:
    using jit_generator = dnnl::impl::cpu::x64::jit_generator;
    ....
   jit_generator& code_generator;
```
Also useful when we decide to change code generator tyep.
Then,

```cpp
template<typename TReg>
class StackAllocator::Reg : public StackAllocator::Address {
...
private:
    static size_t getAlignment() {
        using namespace dnnl::impl::cpu::x64;
        if (std::is_same<TReg, Xbyak::Zmm>::value) {
            return cpu_isa_traits<avx512_core>::vlen;
        } else if (std::is_same<TReg, Xbyak::Ymm>::value) {
            return cpu_isa_traits<avx2>::vlen;
        } else if (std::is_same<TReg, Xbyak::Xmm>::value) {
            return cpu_isa_traits<sse41>::vlen;
        } else {
            return 1;
        }
    }
};

....

inline
void stack_mov(const Xbyak::Xmm& vmm, const StackAllocator::Address& addr) {
    addr.ensureSize(vmm);
    auto& generator = addr.generator();
....
```
So in this way we relatively easy avoid namespace declarations as well as namespace alias declarations in headers.

Reply:
Done. Please, see https://github.com/openvinotoolkit/openvino/pull/13790